.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Once you have installed `nix`, `nix-direnv` and `direnv` this will automatically run `use_flake` for you!

Reply:
Do you know if there's a fallback that only needs `nix` and `direnv`?

Reply:
Nope, would you mind share? 😄

Reply:
According to this we can add it ourselves: https://nixos.wiki/wiki/Flakes

```sh
use_flake() {
  watch_file flake.nix
  watch_file flake.lock
  eval "$(nix print-dev-env --profile "$(direnv_layout_dir)/flake-profile")"
}
```

Reply:
@d4hines do we want this snippet instead of `use flakes`?

Reply:
Yeah, I'd prefer the one with less dependencies if it works equally well.

Reply:
Done